test/datasets/utils_test.py

```diff
@@ -45,3 +47,26 @@ def test_walk_files(self):
             assert found == self.expected[i]
             n_ites += 1
         assert n_ites == len(self.expected)
+
+
+class TestIterator(TorchaudioTestCase):
+    backend = 'default'
+    path = get_asset_path()
+
+    def test_commonvoice(self):
+        data = COMMONVOICE(self.path, url="tatar")
+        data[0]
```

Comment:
You can remove this `test_commonvoice` as the test suite you adding covers more than what this method can do.